### ISSUE TITLE
Task force radio not working outside of the base area

### ISSUE THREAD

**Issue by Kylemeister2** [DUWS-R-Team/DUWS-R#122]
My friend and I have been trying to play this scenario with mods enabled, one of them is task force radio, which works well until we go out of the range of the base or get into vehicles. This is only a problem on this scenario as we've tried it on many others to test and I don't know what's causing it.

**Comment by ultranoobian** [comment]
By out of range, does that also apply to squad radio?

**Comment by Kylemeister2** [comment]
Yeah, it seems like the whole mod stops working when out of range of the base, everyone in the scenario can hear each other normally in teamspeak after leaving the base area whether using the squad radio or not
